Makefile

```diff
@@ -52,8 +54,8 @@ install::
 	mkdir -p "$(PULUMI_NODE_MODULES)/$(NODE_MODULE_NAME)"
 	cp -r ${PACKDIR}/nodejs/bin/. "$(PULUMI_NODE_MODULES)/$(NODE_MODULE_NAME)"
 	rm -rf "$(PULUMI_NODE_MODULES)/$(NODE_MODULE_NAME)/node_modules"
+	cp -r ${PACKDIR}/nodejs/node_modules "$(PULUMI_NODE_MODULES)/$(NODE_MODULE_NAME)"
 	cd "$(PULUMI_NODE_MODULES)/$(NODE_MODULE_NAME)" && \
-		yarn install --offline --production && \
```

Comment:
We don't do this since it will install the set of dependencies listed in package.json, ignoring any linking you've done.

Note: if we *do* want tis behavior (perhaps on travis runs) we could consider a new make target (or something else) to get this done.